app/services/action_builder.rb

```diff
@@ -12,7 +12,7 @@ def build_action(extra_attrs = {})
       subscribed_member: subscribed_member
     }.merge(extra_attrs))
 
-    ActionQueue::Pusher.push(action)
+    ActionQueue::Pusher.push(:new_action, action)
```

Comment:
since almost all actions are new actions, might it make more sense to make the `new_survey_response` flag be an optional second parameter?